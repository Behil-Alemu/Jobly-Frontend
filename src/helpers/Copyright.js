import React from 'react';
import { Typography, Box } from '@mui/material';

function Copyright() {
	return (
		<Box mt={5}>
			<Typography variant="body2" color="textSecondary" align="center">
				{'Copyright Â© '}
				Jobly/
				{''}
				{new Date().getFullYear()}
				{'.'}
			</Typography>
		</Box>
	);
}

export default Copyright;
