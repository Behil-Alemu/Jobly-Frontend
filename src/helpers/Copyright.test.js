import React from "react";
import { render } from "@testing-library/react";
import Copyright from './Copyright'
import { MemoryRouter } from "react-router";


it("renders without crashing", function() {
  render(<Copyright />);
});

it("matches snapshot with copyright", function () {
  const { asFragment } = render(
      <MemoryRouter>
        <Copyright
        />
      </MemoryRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});
