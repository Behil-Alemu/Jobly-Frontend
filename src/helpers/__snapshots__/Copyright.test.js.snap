// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot with copyright 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-y1gt6f"
  >
    <p
      class="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter css-1vl95ex-MuiTypography-root"
    >
      Copyright Â© Jobly/2023.
    </p>
  </div>
</DocumentFragment>
`;
